src/Avalonia.Visuals/Rendering/RenderLoop.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Collections.Generic;
+using Avalonia.Logging;
+using Avalonia.Threading;
+
+namespace Avalonia.Rendering
+{
+    public class RenderLoop : IRenderLoop
+    {
+        private readonly IDispatcher _dispatcher;
+        private List<IRenderLoopTask> _items = new List<IRenderLoopTask>();
+        private IRenderTimer _timer;
+        private volatile bool inTick;
```

Comment:
nit: _inTick
also why is this volatile?

Reply:
Good question! I think it was originally accessed across threads, but that's not longer the case so will remove.